extension/link-options.md

```diff
@@ -1,37 +1,37 @@
 ---
-title: Link Options
-description: Details and options on link generation for sharing traffic captured from the Fiddler Jam extension
+title: Sharing Options
+description: An article about the sharing and security options for submitting a recorded Fiddler Jam logs.
```

Comment:
Provide more value to the meta description by removing unnecessary words and adding more keywords that are related to the content, e.g.:

```suggestion
description: Sharing and security options for recorded Fiddler Jam logs. Share with a link or with specific people, and use an optional password. 
```